sdk/core/azure-core/inc/azure/core/context.hpp

```diff
@@ -55,53 +57,47 @@ namespace Azure { namespace Core {
 
       bool operator!=(Key const& other) const { return !(*this == other); }
     };
-
-    /**
-     * @brief A type used to provide a point in time for context expiration.
-     */
-    using time_point = std::chrono::system_clock::time_point;
-
+    
   private:
     struct ContextSharedState
     {
       std::shared_ptr<ContextSharedState> Parent;
-      std::atomic_int64_t CancelAtMsecSinceEpoch;
+      std::atomic<DateTime::rep> Deadline;
```

Comment:
What's a `::rep`? Is it public?

Reply:
yes - it is inherited from `std::chrono_time_point`